protocol/src/main/java/org/fido/iot/protocol/epid/EpidMaterialService.java

```diff
@@ -79,13 +79,12 @@ public Composite getSigInfo(Composite sigA) throws IOException {
     switch (sgType) {
       case Const.SG_EPIDv10:
       case Const.SG_EPIDv11:
-        Composite certRequestSigInfo =
-            Composite.newArray().set(Const.FIRST_KEY, sgType).set(Const.SECOND_KEY, sigA.toBytes());
 
         byte[] certBytes = Const.EMPTY_BYTE;
         try {
           certBytes = getGroupCertSigma10(sigA);
         } catch (RuntimeException ex) {
+          System.out.println("Runtime Exception in getSigInfo");
```

Comment:
Console print statements will be replaced with proper logging statements in the next iteration.